packages/components/tests/integration/components/hds/form/textarea/field-test.js

```diff
@@ -63,25 +63,29 @@ module('Integration | Component | hds/form/textarea/field', function (hooks) {
       hbs`<Hds::Form::Textarea::Field as |F|>
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
+          <F.CharacterCount>20/40</F.CharacterCount>
           <F.Error>This is the error</F.Error>
         </Hds::Form::Textarea::Field>`
     );
     assert.dom('.hds-form-field__label').exists();
     assert.dom('.hds-form-field__helper-text').exists();
     assert.dom('.hds-form-field__control').exists();
+    assert.dom('.hds-form-field__character-count').exists();
     assert.dom('.hds-form-field__error').exists();
   });
   test('it does not render the yielded contextual components if not provided', async function (assert) {
     await render(hbs`<Hds::Form::Textarea::Field />`);
     assert.dom('.hds-form-field__label').doesNotExist();
     assert.dom('.hds-form-field__helper-text').doesNotExist();
+    assert.dom('.hds-form-field__character-count').doesNotExist();
     assert.dom('.hds-form-field__error').doesNotExist();
   });
   test('it automatically provides all the ID relations between the elements', async function (assert) {
     await render(
       hbs`<Hds::Form::Textarea::Field @extraAriaDescribedBy="extra" as |F|>
           <F.Label>This is the label</F.Label>
           <F.HelperText>This is the helper text</F.HelperText>
+          <F.CharacterCount>20/40</F.CharacterCount>
```

Comment:
same as above, should we test the "dynamic" variant too?

Reply:
updated in 9127638